torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -1898,7 +1898,8 @@ static std::unordered_set<std::string> nodeTypeReliableForTracer = {
     "onnx::Cast",
     "onnx::Constant",
     "onnx::Relu",
-    "com.microsoft::Gelu"};
+    "com.microsoft::Gelu",
+    "aten::ATen"};
```

Comment:
are we sure whether `aten::ATen` should be in `nodeTypeReliableForTracer` in all scenarios? Maybe we need to add a comment saying that this is for backward compatibility (to achieve consistent result in UT)?

Reply:
The PR is intended to keep back compatibility, so adding `aten::ATen` to `nodeTypeReliableForTracer` is necessary for that intent. It is not desired to have shape inference with different outcomes before and after this PR.